standalone_app/src/sqlite3.ts

```diff
@@ -287,30 +287,6 @@ const parseDistributionJSON = (t: string): Distribution => {
       step: parsed.attributes.step as number,
       log: parsed.attributes.log as boolean,
     }
-  } else if (parsed.name === "UniformDistribution") {
```

Comment:
The standalone app does not depend on Optuna directly, so that it seems not relevant with the dropping support for Optuna v2. Could you keep these existing code?

Reply:
oh, sorry, I understand. I will fix it.